### PR TITLE
Fix: Loosing environment variables at StartProcess

### PR DESCRIPTION
**Pull request by thephoenixofthevoid**
Related to #2159

Let's continue discussions here

### CODE REVIEW

File: weed/command/fuse.go [hunk 1]
```diff
@@ -212,9 +219,9 @@ func runFuse(cmd *Command, args []string) bool {
 	}
 
 	// I don't know why PATH environment variable is lost
```
Comment (chrislusf) [inline weed/command/fuse.go]: clean up the comments?

File: weed/command/fuse.go [hunk 2]
```diff
@@ -195,7 +195,14 @@ func runFuse(cmd *Command, args []string) bool {
 		arg0 := os.Args[0]
 		argv := append(os.Args, "-o", "child")
 
-		attr := os.ProcAttr{}
+		attr := os.ProcAttr{}		
+		attr.Env = os.Environ() 
+		// By default attr.Env is [] ie empty set of env vars
+		// not nil and not os.Environ()
+		
+		//attr.Dir = "."
```
Comment (chrislusf) [inline weed/command/fuse.go]: Are these needed? If not, please clean up.

Comment (thephoenixofthevoid) [reply weed/command/fuse.go]: Including line 204 or not? Line 204 allows for enabling output from child process to check what envs are passed (literally none (zero) were passed). I will leave it here:
```
attr.Files = []*os.File{ nil, os.Stdout, os.Stderr }
```
and remove in the code.

Comment (thephoenixofthevoid) [reply weed/command/fuse.go]: Also will remove these:
```
// I don't know why PATH environment variable is lost
// if err := os.Setenv("PATH", "/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin"); err != nil {
//	panic(fmt.Errorf("setenv: %s", err))
// }
```

**Pull request merged**